jupyter_server/base/handlers.py

```diff
@@ -192,7 +192,17 @@ def login_available(self):
         return bool(self.login_handler.get_login_available(self.settings))
 
 
-class JupyterHandler(AuthenticatedHandler):
+class AuthorizedHandlerMixin:
+    """A mixin class for Tornado request handlers that checks whether
+    the current user is authorized to execute the current action.
+    """
+
+    def user_is_authorized(self, user, action, resource):
```

Comment:
I cannot find any use/mention of this method anywhere, so I question if this Mixin is needed at all?

Reply:
You're right. This is an old relic of a previous version of this PR. With the `Authorizer` class, this mixin isn't needed anymore.

Reply:
Or maybe instead, this chunk should be moved into the Mixin class:
```python
    @property
    def authorizer(self):
        return self.settings["authorizer"]
```